package.json

```diff
@@ -9,7 +9,7 @@
     "node": ">=8.12.0"
   },
   "scripts": {
-    "seed-database": "node seedData.js",
+    "seed-database": "mysql -u root -p < schema.sql",
```

Comment:
consider using environment vars since you set that up originally . I believe this would be `mysql -u %DB_HOST% .. ` but googling would confirm.

Reply:
I like the env variables for sure, thanks!  For this line though, then someone installing and using this repo would have to create their own `.env` file settings just to seed the DB, which is prolly not a big deal, but I was thinking this saves them a step?

Reply:
fair. hopefully users always have root authorized, and this prompts for password otherwise so lets consider this moot for this exercise.